### PR TITLE
chore(content-manager): migrate bulk publish and unpublish to v5

### PR DESCRIPTION
**Pull request by Feranchz**
### What does it do?

Start using the new documents service in the content manager for bulk publish and bulk unpublish.

### How to test it?

Try to bulkPublish/unpublish entities directly with the API.

### CODE REVIEW

**Comment by joshuaellis** [comment]
Are you planning on doing the FE work on this branch and work together, or do separate PRs? benefits of the former, is Madhuri can accurately validate your implementations now as opposed to later.

File: packages/core/content-manager/server/src/services/document-manager.ts [hunk 1]
```diff
@@ -197,84 +177,35 @@ const documentManager = ({ strapi }: { strapi: Core.Strapi }) => {
...
-        data,
-      });
-      // Get the updated entities since updateMany only returns the count
-      const publishedEntities = await strapi.db.query(uid).findMany({
-        where: filters,
-        populate,
+      const publishedEntities = await strapi.db.transaction(async () => {
+        return async.map(entities, async ({ documentId }: Document) => {
+          // @ts-expect-error - documents.publish is not accepting populate but it should
+          const publishedEntity = await strapi.documents(uid).publish({ documentId, populate });
```
Comment (Marc-Roig) [inline packages/core/content-manager/server/src/services/document-manager.ts:185]: Is the populate necessary in this situacion? we don't need to return the fully populated entries 
(We still need to work on accepting populate on the doc service publish though)

**Comment by Marc-Roig** [comment]
Did we have any bulk publish api tests before? (they might have been skipped when developving v5)

Could you enable them again (or add some if they don't exist)?

File: packages/core/content-manager/server/src/services/document-manager.ts (hunk 1 above)
Comment (Feranchz) [reply packages/core/content-manager/server/src/services/document-manager.ts:185]: I was checking how works the publish function on the documents service and I guess is not necessary 🤔. In v4 the populate was necessary to make validations but here is not, so I guess is something we can save here

**Comment by Feranchz** [comment]
> Are you planning on doing the FE work on this branch and work together, or do separate PRs? benefits of the former, is Madhuri can accurately validate your implementations now as opposed to later.

@joshuaellis That's a good point. Lets use this branch as a "feature branch" for now but keeping smaller PRs to merge things here. @madhurisandbhor For now is only implemented the bulkPublish/unpublish but you can start using them and create a PR aiming this branch, then we merge all together

File: packages/core/content-manager/server/src/services/document-manager.ts [hunk 2]
```diff
@@ -1,41 +1,21 @@
...
 export type Document = Modules.Documents.Result<UID.ContentType>;
 
 const { ApplicationError } = errors;
-const { ENTRY_PUBLISH, ENTRY_UNPUBLISH } = ALLOWED_WEBHOOK_EVENTS;
 const { PUBLISHED_AT_ATTRIBUTE } = contentTypes.constants;
 
 const omitPublishedAtField = omit(PUBLISHED_AT_ATTRIBUTE);
 const omitIdField = omit('id');
 
-const emitEvent = async (uid: UID.ContentType, event: string, document: Document) => {
```
Comment (remidej) [inline packages/core/content-manager/server/src/services/document-manager.ts:21]: I wonder what happened to the emitted events?

**Pull request closed without merging**